global/ramda.d.ts

```diff
@@ -1222,8 +1225,8 @@ declare namespace R {
          * Returns a function that when supplied an object returns the indicated property of that object, if it exists.
          * Note: TS1.9 # replace any by dictionary
          */
-        prop<T>(p: string, obj: any): T;
-        prop<T>(p: string): <T>(obj: any) => T;
+        prop<T>(p: string | number, obj: any): T;
```

Comment:
The signature of prop on the ramda site says `s → {s: a} → a | Undefined`. So why would you add Number too?